ext/curl/tests/curl_pushfunction_trampoline.phpt

```diff
@@ -2,7 +2,7 @@
 Test trampoline for curl option CURLMOPT_PUSHFUNCTION
 --EXTENSIONS--
 curl
---XLEAK--
+--XFAIL--
```

Comment:
Upgrading to Ubuntu (and curl 8.5.0) changes behavior of this test. Namely, it outputs:

> Trampoline for trampoline

@Girgias Can you make sense of this? Why isn't the callback called before?

Reply:
It recently started leaking, which is when I added the xleak. Hence, I'm pretty sure it wasn't skipped.

Reply:
Huuuuum, I'll have a look soon, maybe @Ayesh has an idea?